tests/unit_node/assert_test.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import * as assert from "node:assert";
+
+Deno.test("[node/assert] .throws() compares Error instance", () => {
+  assert.throws(
```

Comment:
Just calling it like this would throw an error previously, see the linked issue.